src/data-access/ItemSearcher.ts

```diff
@@ -0,0 +1,19 @@
+export interface SearchedItemOption {
+	itemId: string;
```

Comment:
I’m fine with renaming it, but I don’t think it would reduce the amount of “work”, since I’d still put something like `id: result.id` in `processResponse`. (I wouldn’t just return the response directly, even though, with the `display` changes merged, it should be compatible.)

Reply:
ok, then let's keep it as it is